messages/es.json

```diff
@@ -15,6 +15,10 @@
   "admin/messages.cancel": "Cancelar",
   "admin/messages.retry": "Procesar de nuevo",
   "admin/messages.done": "Finalizar",
+  "admin/messages.catalog.download.instruction1": "1. Descarga los datos que quieres traducir de tu catálogo",
+  "admin/messages.catalog.download.instruction2": "2. Abra el archivo exportado en Excel o Números",
```

Comment:
I don't think `Numbers` should be translated into `Números`, since it's the name of the software.